src/schema/submission_centric_schema.yaml

```diff
@@ -216,7 +218,8 @@ classes:
         description: >-
           One or more attributes that further characterizes this Library Preparation Protocol.
       target regions:
-        required: true
+        required: false
+        recommended: true
```

Comment:
Why was it recommended? From the sequencing perspective, it contains hundreds of sequences, and we have the feedback that it is not feasible to capture. It will be discussed in the scope of version 0.10

Reply:
Agree, I'll delete it in this case.